src/detect-flowbits.c

```diff
@@ -287,6 +363,13 @@ int DetectFlowbitSetup (DetectEngineCtx *de_ctx, Signature *s, const char *rawst
     return 0;
 
 error:
+
```

Comment:
Pre-existing error -- `cd->or_list` will be leaked it `SigMatchAlloc()` fails.

Suggest calling `DetectFlowbitFree(cd)` instead of `SCFree(cd)`.